templates/bootstrap/features.tmpl

```diff
@@ -134,9 +134,9 @@ this.Then(/^<%= step.name.replace(/"[^"]*"/g, '"\(\[\^\"\]\*\)"') %>$/, function
                       <% } %>
 
                        <% if (step.image) { %>
-                                                <a class="toggle" href="#">Screenshot -</a>
+                                                <a class="toggle" href="javascript: void(0);">Screenshot -</a>
```

Comment:
Could you change the toggle from `-` to `+` at `Screenshot -` so that Screenshot should launch at the first click? Otherwise, the user has to click twice initially.